kubeshop/botkube-docs
# Fix glitch with displaying cookie banner for a split second

## Description

Changes proposed in this pull request:

- Simplify logic for embedded mode
   - we don't use it anywhere else apart from cookie banner, so let's keep it simple.
- Fix glitch with displaying cookie banner for a split second

## Testing

See how it behaved until now:
1. Navigate to e.g. https://docs.botkube.io/installation/slack/?embedded=true in the incognito mode.
1. Try to refresh the page multiple times. See the cookie banner for a split second every time.

1. Navigate to https://d568c9cb.botkube-docs-dcb.pages.dev/installation/slack/?embedded=true in the incognito mode
1. Try to refresh the page multiple times. There cookie banner won't be displayed for first 100-200 ms or so.
